seacrowd/sea_datasets/cc_aligned_doc/cc_aligned_doc.py

```diff
@@ -68,19 +68,20 @@ class CCAlignedDocDataset(datasets.GeneratorBasedBuilder):
 
     SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
     SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+    SEACROWD_SCHEMA_NAME = "t2t"
 
     BUILDER_CONFIGS = [SEACrowdConfig(name=f"{_DATASETNAME}_{subset}_source", version=datasets.Version(_SOURCE_VERSION), description=f"{_DATASETNAME} source schema", schema="source", subset_id=f"{_DATASETNAME}",) for subset in _SUBSETS.keys()] + [
         SEACrowdConfig(
-            name=f"{_DATASETNAME}_{subset}_seacrowd_t2t",
+            name=f"{_DATASETNAME}_{subset}_seacrowd_{schema_name}",
             version=datasets.Version(_SEACROWD_VERSION),
             description=f"{_DATASETNAME} SEACrowd schema",
-            schema="seacrowd_t2t",
+            schema=f"seacrowd_{schema_name}",
             subset_id=f"{_DATASETNAME}",
         )
-        for subset in _SUBSETS.keys()
+        for subset, schema_name in zip(_SUBSETS.keys(), len(_SUBSETS.keys()) * [SEACROWD_SCHEMA_NAME])
```

Comment:
I could not just use SEACROWD_SCHEMA_NAME inside this loop, error that it is not defined occurs. So I decided to do it this way :)